api/handlers.go

```diff
@@ -910,6 +1283,14 @@ func appendListMods(mods *[]qm.QueryMod, r ListRequest) (int, int) {
 	return limit, offset
 }
 
+func appendQueryFilter(mods *[]qm.QueryMod, r QueryFilter, column string) {
+	if r.Query == "" {
+		return
+	}
+	q := fmt.Sprintf("%s ILIKE '%%%s%%'", column, r.Query)
```

Comment:
this is vulnerable to sql injection.
Should be 
```
q := qm.Where(fmt.Sprintf("%s ILIKE ?", column), r.Query)
```
so that sqlboiler can sanitize `r.Query` before it goes to the DB

Reply:
replaced